docs/clusters/santis.md

```diff
@@ -1,3 +1,127 @@
 [](){#ref-cluster-santis}
 # Santis
 
+Santis is an Alps cluster that provides GPU accelerators and file systems designed to meet the needs of climate and weather models for the [CWp][ref-platform-cwp].
```

Comment:
Mostly a question for myself to know: is CWp/MLp/etc. the way we capitalize this, not CWP/MLP/etc.

Reply:
I just asked, and it was `XYp`, but it was changed to `XYP`.
I will make the change.